source/base/quadrature.cc

```diff
@@ -70,8 +70,7 @@ Quadrature<dim>::initialize(const ArrayView<const Point<dim>> &points,
       this->weights.insert(this->weights.end(), weights.begin(), weights.end());
     }
   else
-    this->weights.resize(points.size(),
-                         std::numeric_limits<double>::infinity());
+    this->weights.resize(points.size(), numbers::signaling_nan<double>());
```

Comment:
```suggestion
    this->weights.resize(points.size(), numbers::numeric_limits<double>::signaling_NaN());
```

Reply:
That comes out to the same, doesn't it? We tend to use `numbers::` more often:
```
1/dealii> grep -r signaling_ include/ source/ | grep limits | wc -l
11
1/dealii> grep -r signaling_ include/ source/ | grep numbers:: | wc -l
50
```
I don't feel strongly either way.

Reply:
Apologies, I misread and missed that this is our own `numbers::` namespace.